src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -500,3 +500,47 @@ def write_to_disk(data: pd.DataFrame, path: Path):
             f"Supported extensions are {metadata.SUPPORTED_EXTENSIONS}. "
             f"{path.suffix[1:]} was provided."
         )
+
+
+def copy_from_household_member(
+    pop: pd.DataFrame, randomness_stream: RandomnessStream
+) -> pd.DataFrame:
+    # Creates copy_age, copy_date_of_birth, and copy_ssn from household members
+    # Note: copy value can be original value but copies from another household member
+
+    copy_cols = metadata.COPY_HOUSEHOLD_MEMBER_COLS
+    for col in [column for column in copy_cols.keys() if column in pop.columns]:
+        pop[copy_cols[col]] = np.nan
+        if col == "has_ssn":
+            # Subset to rows that have SSN so we do not try and map nans.
+            households = (
+                pop[pop[col]].groupby("household_id").apply(lambda df_g: list(df_g.index))
+            )
+        else:
+            # This makes the assumption age and date of birth will not have nans at this poitn
```

Comment:
Is this actually an assumption at this point? Do you have reason to believe this isn't the case? I honestly don't remember

Reply:
This comment is basically stating that since this our population state table, age and date of birth should not have nans as we do not have anything in the simulation that would create nans in those columns. It is ultimately a reminder that if for some reason we change that in the future we would need to subset our groupby object like we do in 517.